content/en/docs/concepts/security/security-checklist.md

```diff
@@ -206,6 +206,24 @@ SELinux is only available on Linux nodes, and enabled in
 [some Linux distributions](https://en.wikipedia.org/wiki/Security-Enhanced_Linux#Implementations).
 {{< /note >}}
 
+## Logs and auditing
+
+- [ ] Audit logs, if enabled, are protected from general access.
+- [ ] The `/logs` API is disabled (you are running kube-apiserver with
+  `--enable-logs-handler=false`).
+
+  Kubernetes includes a deprecated `/logs` API endpoint, enabled by default,
```

Comment:
```suggestion
  Kubernetes includes a `/logs` API endpoint, enabled by default,
```

Reply:
(I think) this is because we might undeprecate the endpoint.

Reply:
yeah, possibly